charts/ontology/templates/service.yaml

```
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Chart.Name }}
+  labels:
+{{ include "ontology.labels" . | indent 4 }}
+spec:
+  ports:
+    - name: http
+      port: 8080
```

```suggestion
  ports:
    - name: http
      protocol: TCP
      port: 80
    - name: https
      protocol: TCP
      port: 443
```
Now that the proxy is up and running I think the service should communicate with those ports rather than directly with the ontology container

---

nevermind, I misread the chart on this one.